src/Oro/Bundle/DataGridBundle/Resources/public/js/datagrid/page-size.js

```diff
@@ -110,7 +110,7 @@ define([
                 _.bind(
                     function(item) {
                         return item.size === undefined ?
-                            this.collection.state.pageSize === item : this.collection.state.pageSize === item.size;
+                            this.collection.state.pageSize == item : this.collection.state.pageSize == item.size;
                     },
```

Comment:
This solution does not fit requirements of JS code style. Abstract Equality Comparison is restricted by adopted JSCS setup.